plugins/woocommerce/src/Admin/Features/Blueprint/RestApi.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+declare( strict_types = 1);
+
+namespace Automattic\WooCommerce\Admin\Features\Blueprint;
+
+use Automattic\WooCommerce\Blueprint\ExportSchema;
+use Automattic\WooCommerce\Blueprint\ImportSchema;
+use Automattic\WooCommerce\Blueprint\JsonResultFormatter;
+use Automattic\WooCommerce\Blueprint\ZipExportedSchema;
+
+/**
+ * Class RestApi
+ *
+ * This class handles the REST API endpoints for importing and exporting WooCommerce Blueprints.
+ *
+ * @package Automattic\WooCommerce\Admin\Features\Blueprint
+ */
+class RestApi {
+	/**
+	 * Endpoint namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace = 'blueprint';
+
+	/**
+	 * Register routes.
+	 *
+	 * @since 3.5.0
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/import',
+			array(
+				array(
+					'methods'             => \WP_REST_Server::CREATABLE,
+					'callback'            => array( $this, 'import' ),
+					'permission_callback' => array( $this, 'check_permission' ),
+				),
+			)
+		);
+
+		register_rest_route(
+			$this->namespace,
+			'/export',
+			array(
+				array(
+					'methods'             => \WP_REST_Server::CREATABLE,
+					'callback'            => array( $this, 'export' ),
+					'permission_callback' => array( $this, 'check_permission' ),
+					'args'                => array(
+						'steps'         => array(
+							'description'       => 'A list of plugins to install',
+							'type'              => 'array',
+							'items'             => 'string',
+							'default'           => array(),
+							'sanitize_callback' => function ( $value ) {
+								return array_map(
+									function ( $value ) {
+										return sanitize_text_field( $value );
+									},
+									$value
+								);
+							},
+							'required'          => false,
+						),
+						'export_as_zip' => array(
+							'description' => 'Export as a zip file',
+							'type'        => 'boolean',
+							'default'     => false,
+							'required'    => false,
+						),
+					),
+				),
+			)
+		);
+	}
+
+	public function check_permission() {
+		if ( ! wc_rest_check_manager_permissions( 'settings', 'read' ) ) {
+			return new \WP_Error( 'woocommerce_rest_cannot_view', __( 'Sorry, you cannot list resources.', 'woocommerce' ), array( 'status' => rest_authorization_required_code() ) );
+		}
+
+		return true;
+	}
+
+	/**
+	 * Handle the export request.
+	 *
+	 * @param \WP_REST_Request $request The request object.
+	 * @return \WP_HTTP_Response The response object.
+	 */
+	public function export( $request ) {
+		$steps         = $request->get_param( 'steps', array() );
+		$export_as_zip = $request->get_param( 'export_as_zip', false );
```

Comment:
Does the `get_param` method in WP_REST_Request accept a second argument for a default value? 

https://developer.wordpress.org/reference/classes/wp_rest_request/get_param/

Reply:
Updated in [Remove unavailable 2nd argument from get_param -- default values are …](https://github.com/woocommerce/woocommerce/pull/49765/commits/eb654cbcf574e235eb4214cb39adde9f73b6390b)